pennylane/tape/qscript.py

```diff
@@ -577,8 +576,8 @@ def trainable_params(self) -> list[int]:
         to compute the Jacobian; parameters not marked as trainable will be
         automatically excluded from the Jacobian computation.
 
-        The number of trainable parameters determines the number of parameters passed to
-        :meth:`~.set_parameters`, and changes the default output size of method :meth:`~.get_parameters()`.
+        The number of trainable parameters determines the number of parameters
+        changes the default output size of method :meth:`~.get_parameters()`.
```

Comment:
Right? :sweat_smile: 
```suggestion
        The number of trainable parameters changes the default output size of method :meth:`~.get_parameters()`.
```